packages/ui/src/theme/createTheme.ts

```diff
@@ -43,16 +43,7 @@ function setupTokens(obj: any, path = []) {
     for (const name in obj) {
       if (obj.hasOwnProperty(name)) {
         if (typeof obj[name] !== 'object') {
-          // If we get to this point that means there is a 'dangling' part of the theme object
-          // basically some part of the theme object that is not a design token, which is
-          // anything that is not an object with a value attribute
-          console.warn(
-            `Non-design token found when creating the theme at path: ${path.join(
-              '.'
-            )}.${name}\nDid you forget to add '{value:"${obj[name]}"}'?`
-          );
-          // Keep the users data there just in case
-          tokens[name] = obj[name];
+          tokens[name] = setupTokens({ value: obj[name] }, path.concat(name));
```

Comment:
It's so simple.

Reply:
TBH I love this. I've been thinking about it for 30 mins and it just works.

Reply:
The TS is strong with this one.